spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -16,19 +16,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+[[release_notes_31]]
+== Release Notes for Microprofile OpenAPI 3.1
```

Comment:
```suggestion
== Release Notes for MicroProfile OpenAPI 3.1
```

Reply:
Will fix this here and in the other headings